modules/spacecraftinstruments/rendering/renderableshadowcylinder.cpp

```diff
@@ -210,6 +206,10 @@ RenderableShadowCylinder::RenderableShadowCylinder(const ghoul::Dictionary& dict
         "RenderableShadowCylinder"
     );
 
+    _opacity.setValue(0.25f);
```

Comment:
Are we sure that we want to always set the opacity to 0.25?

Reply:
No, I guess it's better to set the value in the asset files ^^'